commerce_coordinator/apps/commercetools/data.py

```diff
@@ -111,7 +111,7 @@ def order_from_commercetools(order: CTOrder, customer: CTCustomer) -> LegacyOrde
         date_placed=order.last_modified_at,
         total_excl_tax=typed_money_to_string(order.total_price, money_as_decimal_string=SEND_MONEY_AS_DECIMAL_STRING),
         # in dev systems, this isn't set... so let's use UUID, otherwise, let's rely on order number
-        number=order.id,  # Long-term: this_or(order.order_number, order.id),
+        number=order.order_number or order.id,
```

Comment:
Please remove the comment above.
```suggestion
        number=order.order_number,
```